src/components/views/elements/Slider.tsx

```diff
@@ -83,18 +89,73 @@ export default class Slider extends React.Component<IProps> {
 
         let selection = null;
 
+        const onDrag = (e: React.DragEvent) => {
```

Comment:
There's a lot of logic overlap on `onDrag` and `onDragEnd` could we consolidate those ?

Reply:
Even I had that in my mind. Will make a separate function for common operations.